lib/src/st2110/st_header.h

```diff
@@ -146,7 +146,7 @@ struct st_frame_trans {
 
 /* timing for pacing */
 struct st_tx_video_pacing {
-  double trs;         /* in ns for of 2 consecutive packets, T-Frame / N-Packets */
+  uint64_t trs;       /* in ns for of 2 consecutive packets, T-Frame / N-Packets */
```

Comment:
video_session: fix calculation error caused by double precision type.

Found that the inter-packet time always greater than the theoretical value,
5855ns vs 5832ns (1080p 50fps 8-bit). Will cause number of VRX to be negative.

Replace the double type with integer type of time variables. The issue is fixed.

Reply:
Sounds weird. But can you move this change a new PR?

Reply:
Done. I will submit the fix after this PR is merged.

> Sounds weird. But can you move this change a new PR?